SPECS/sm.spec

```diff
@@ -398,49 +295,46 @@ cp -r htmlcov %{buildroot}/htmlcov
 /etc/xapi.d/plugins/linstor-manager
 /opt/xensource/sm/CephFSSR
 /opt/xensource/sm/CephFSSR.py
-/opt/xensource/sm/CephFSSR.pyc
-/opt/xensource/sm/CephFSSR.pyo
 /opt/xensource/sm/EXT4SR
 /opt/xensource/sm/EXT4SR.py
-/opt/xensource/sm/EXT4SR.pyc
-/opt/xensource/sm/EXT4SR.pyo
 /opt/xensource/sm/GlusterFSSR
 /opt/xensource/sm/GlusterFSSR.py
-/opt/xensource/sm/GlusterFSSR.pyc
-/opt/xensource/sm/GlusterFSSR.pyo
 /opt/xensource/sm/LinstorSR
 /opt/xensource/sm/LinstorSR.py
-/opt/xensource/sm/LinstorSR.pyc
-/opt/xensource/sm/LinstorSR.pyo
 /opt/xensource/sm/MooseFSSR
 /opt/xensource/sm/MooseFSSR.py
-/opt/xensource/sm/MooseFSSR.pyc
-/opt/xensource/sm/MooseFSSR.pyo
 /opt/xensource/sm/XFSSR
 /opt/xensource/sm/XFSSR.py
-/opt/xensource/sm/XFSSR.pyc
-/opt/xensource/sm/XFSSR.pyo
 /opt/xensource/sm/ZFSSR
 /opt/xensource/sm/ZFSSR.py
-/opt/xensource/sm/ZFSSR.pyc
-/opt/xensource/sm/ZFSSR.pyo
 /opt/xensource/sm/linstorjournaler.py
-/opt/xensource/sm/linstorjournaler.pyc
-/opt/xensource/sm/linstorjournaler.pyo
 /opt/xensource/sm/linstorvhdutil.py
-/opt/xensource/sm/linstorvhdutil.pyc
-/opt/xensource/sm/linstorvhdutil.pyo
 /opt/xensource/sm/linstorvolumemanager.py
-/opt/xensource/sm/linstorvolumemanager.pyc
-/opt/xensource/sm/linstorvolumemanager.pyo
 /opt/xensource/libexec/linstor-monitord
 %{_unitdir}/linstor-monitor.service
 
 %changelog
-* Thu Dec 15 2022 Ronan Abhamon <ronan.abhamon@vates.fr> - 2.46.16-1.1
-- Rebase on CH 8.3 Preview
-- Remove patches merged upstream
-- Keep other patches still necessary.
+* Wed Mar 08 2023 Yann Dirson <yann.dirson@vates.fr> - 3.0.3-1.0.yd1
+- patches for python3 compat
+- don't run pylint at build time, remove BuildRequires: python3-pylint
+- use python36-bitarray not python3-bitarray
+- BuildRequires: python36-mock for %check
+- from Ronan Abhamon <ronan.abhamon@vates.fr>
+  - Rebase on CH 8.3 Preview
+  - Remove patches merged upstream
+  - Keep other patches still necessary.
```

Comment:
The definition of changelog entries is rather strict: each line should start with dash, without indentation.

See https://rpm-packaging-guide.github.io/

Reply:
Hm, this guide does not seem to be a standards document, but rather a didactic document where not every subject is meant to be exposed.  What's more standards-looking seems to live in https://rpm-software-management.github.io/rpm/manual/spec.html.  It precisely has an empty section for `%changelog`, so I took the liberty of adapting the Debian way of doing it, and the tools did not complain, even rpmlint.

Reply:
Ok, maybe I'm biased by habits, as this is not common.